### PR TITLE
PLAT-182: all insolard CLI commands

### PR DESCRIPTION
**Pull request by AndreyBronin**
**- What I did**

- descriptions and correct flags  for CLI commands
- readme update, run node guide in production mode

### CODE REVIEW

File: ledger-core/v2/README.md [hunk 1]
```diff
@@ -71,6 +71,30 @@ To test Insolar locally, install it and deploy as described below.
...
      * `-r`: Number of transfer requests to be sent in each thread.
 
+## Run single node
+
+We use two-process approach for running node in production.
+Time-critical consensus and network algorithms runs in the first process with high priority and resources quota.
+The other node components run in the second process.
+
+You have two options to run Insolar node in production:
+ * first process manage the second
```
Comment (ivanshibitov) [inline ledger-core/v2/README.md:81]: manageS

File: ledger-core/v2/README.md [hunk 2]
```diff
@@ -37,41 +52,81 @@ To test Insolar locally, install it and deploy as described below.
...
+You have two options to run Insolar node in production:
+
+ * First process manages the second.
+ 
+ * A watchdog manages each process separately.
+ 
+In case of the first option, you can use a systemd service script:
+
+```
+insolard config generate --role=virtual -c ./node.conf
```
Comment (aalekseev-ru) [inline ledger-core/v2/README.md:116]: I tried to run it on the branch and get an error
aalekseev@MacBook-Pro-4 v2 % ./bin/insolard config generate --role=virtual -c ./node.conf 
Error: unknown flag: --role

Comment (aalekseev-ru) [reply ledger-core/v2/README.md:116]: well, I forgot to rebuild bin -_-
but after that, I also get an error

./bin/insolard config generate --role=virtual -c ./node.conf
{"level":"info","writeDuration":"3.000µs","message":"TODO: generate configuration with config=./node.conf role=virtual mode="}
{"level":"fatal","stack":"main.configGenerateCommand.func1\n\t/Users/aalekseev/go/src/github.com/insolar/assured-ledger/ledger-core/v2/cmd/insolard/commands.go:169","writeDuration":"0.000µs","message":"not implemented"}

is it ok?

Comment (AndreyBronin) [reply ledger-core/v2/README.md:116]: Yes, it's ok, coz this PR is only about CLI commands. Most of them still not implemented

File: ledger-core/v2/README.md [hunk 3]
```diff
@@ -1,34 +1,49 @@
...
 
-3. Go to the package directory:
+2. Go to the package directory:
 
    ```
    cd $GOPATH/src/github.com/insolar/assured-ledger/ledger-core/v2
    ```
 
-4. Install dependencies and build binaries:
+3. Install dependencies and build binaries using [the makefile](https://github.com/insolar/assured-ledger/blob/PLAT-182-all-insolard-cli-commands/ledger-core/v2/Makefile) that automates this process:
```
Comment (sexyhamster) [inline ledger-core/v2/README.md:46]: I wonder do we still need this step? Go will download all necessary packages during build step, make vendor is not needed.

Comment (AndreyBronin) [reply ledger-core/v2/README.md:46]: @sexyhamster *make vendor* step is still needed

**Pull request merged**